controller/CMakeLists.txt

```diff
@@ -5,13 +5,17 @@ set(BEEROCKS_WLAN1_IFACE "wlan0")
 set(BEEROCKS_WLAN2_IFACE "wlan2")
 
 # Platform specific flags
-if (TARGET_PLATFORM STREQUAL "ugw")
-    add_definitions(-DBEEROCKS_UGW)
+if (TARGET_PLATFORM STREQUAL "openwrt")
+    # UGW
+    if (TARGET_PLATFORM_TYPE STREQUAL "ugw")
+        add_definitions(-DBEEROCKS_UGW)
+    # Turris Omnia
+    elseif (TARGET_PLATFORM_TYPE STREQUAL "turris-omnia")
+        set(BEEROCKS_WLAN2_IFACE "wlan1")
```

Comment:
Did we come to a conclusion about this? I suggest we move the interface names to the platform DB, but that of course can be done in a follow-up PR.